tests/unit/test_locationcodes.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for strtools.py"""
+
+import time
+import timeit
+import random
+import importlib
+
+import pytest
+from scipy import stats
+
+import zolltools.nemsis.locationcodes as locationcodes
+
+@pytest.mark.slow
+def test_get_mapping_performance():
+    """
+    measuring performance of get_mapping
+    
+    Takes a sample of the execution time for the first and successive calls to
+    `get_mapping` and determines that successive calls are at least
+    `min_exp_speedup` times faster than the first calls within a certain
+    confidence interval (see `alpha` and comparison to p-value in `assert`
+    statement).
+    
+    `min_exp_speedup` was determined with preliminary testing. See gh 74
+    """
+
+    alpha = 0.05             # confidence threshold
+    min_exp_speedup = 40     # successive reads should be min_exp_speedup times faster
+    num_data_points = 10_000
```

Comment:
min_exp_speedup has been adjusted to 10, because an order of magnitude increase is enough to demonstrate a significant change in how the data is being read (from storage vs. from heap).

num_data_points, now 10 is sufficient, but 100 felt more proper. Set to 100.